recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   url: https://github.com/Galsim-Developers/galsim/archive/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - 0001-remove-run-install-checks.patch
+     - 0001-patch-setup-conda-fftw-for-cross.patch
```

Comment:
Do we want to limit this patch to just osx-arm64? It is correct for every platform so there is no need per say.